server/sonar-web/src/main/js/apps/issues/controller.js

```diff
@@ -5,7 +5,8 @@ define([
 ], function (Controller, ComponentViewer, HomeView) {
 
   var $ = jQuery,
-      EXTRA_FIELDS = 'actions,transitions,actionPlanName',
+      FIELDS = 'component,componentId,project,subProject,rule,status,resolution,author,reporter,assignee,debt,line,message,severity,' +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The line contains 135 characters which is greater than 120 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ALineLength)